src/ci/github-actions/jobs.yml

```diff
@@ -51,6 +51,7 @@ runners:
     # Free some disk space to avoid running out of space during the build.
     free_disk: true
     os: ubuntu-22.04-arm
+    <<: *base-job
```

Comment:
this is missing in job-aarch64-linux-8c too

Reply:
Interesting that it didn't crash. Thanks!